language-configuration.json

```diff
@@ -24,8 +24,8 @@
     ["\"", "\""]
   ],
   "indentationRules": {
-    "increaseIndentPattern": "^\\s*(data|proc|%macro)\\b.*;$",
-    "decreaseIndentPattern": "^\\s*(run;|quit;|%mend;)$"
+    "increaseIndentPattern": "(data|proc|%macro)\\b[^;]*;(\\s|/\\*.*\\*/|\\*[^;]*;)*$",
```

Comment:
it would be great to make the regex case insensitive so that it won't need server change in most cases.
it can accept
```
{
    "pattern": "",
    "flags": ""
}
```